SPECS/initscripts.spec

```diff
@@ -32,7 +32,6 @@ Requires: system-release
 Requires: udev >= 125-1
 Requires: cpio
 Requires: hostname
-Requires: setup >= 2.8.72
```

Comment:
I like to keep it visible and commented, with an "XCP-ng" comment above: it helps in conflict resolution and understanding of diffs between our spec file and theirs.